tests/TestHelper.php

```diff
@@ -34,10 +34,7 @@
 }
 
 $phpunitVersion = PHPUnit_Runner_Version::id();
-if ($phpunitVersion == '@' . 'package_version@' || !version_compare($phpunitVersion, '3.8', '<=')) {
-    echo "This version of PHPUnit is not supported.";
-    exit(1);
-} elseif (version_compare($phpunitVersion, '3.5.0', '>=')) {
+if (version_compare($phpunitVersion, '3.5.0', '>=')) {
```

Comment:
Ah this patch might have been neater than what I did in 593cd3f8d5940b945514a55f4d699fe44ecadb79